conda_env/specs/jinja2_file.py

```diff
@@ -0,0 +1,36 @@
+try:
+    import jinja2
+except ImportError:
+    jinja2 = None
+from .. import env
+from ..exceptions import EnvironmentFileNotFound, Jinja2NotInstalled
+
+
+class Jinja2Spec(object):
+    _environment = None
```

Comment:
because https://github.com/conda/conda-env/blob/jinja2/conda_env/specs/jinja2_file.py#L34
I want to be sure it's been defined already